static/js/notifications.js

```diff
@@ -415,6 +419,12 @@ export function should_send_desktop_notification(message) {
 }
 
 export function should_send_audible_notification(message) {
+    // If `no sound` is selected as the notification sound, send
+    // no audible notification.
+    if ($("#notification-sound-audio").prop("disabled")) {
+        return false;
+    }
```

Comment:
I would prefer a check based on `page_params.notification_sound`, not the state of the DOM, since that seems more like the source of truth here.